certif/app/styles/pages/authenticated/sessions/import.scss

```
@@ -101,24 +96,27 @@
     font-weight: 500;
   }
 
+  &__section--download-button .download-buttons {
```

L'harmonisation de la taille des boutons `[Télécharger (.csv)]` et `[Importer (.csv)]` a été cassé, à cause du fait que la classe `download-buttons` n'est pas injecté dans le bon élément du composant `PixButtonUpload`. Il est injecté dans l'`input` et non pas dans le `label` et ça ne marche pas, certainement dû à la gestion du container du composant.

```suggestion
  &__section--download-button .download-buttons,  &__section--download-button .pix-button-upload {
```